targets/TARGET_STM/TARGET_STM32G0/STM32Cube_FW/CMSIS/stm32g0b1xx.h

```diff
@@ -104,7 +104,11 @@ typedef enum
   USART1_IRQn                 = 27,     /*!< USART1 Interrupt                                                  */
   USART2_LPUART2_IRQn         = 28,     /*!< USART2 + LPUART2 Interrupt                                        */
   USART3_4_5_6_LPUART1_IRQn   = 29,     /*!< USART3, USART4, USART5, USART6, LPUART1 globlal Interrupts (combined with EXTI 28) */
-  CEC_IRQn                    = 30,     /*!< CEC Interrupt(combined with EXTI 27)                               */
+  CEC_IRQn                    = 30,     /*!< CEC Interrupt(combined with EXTI 27)                              */
+  FDCAN1_IT0_IRQn             = 31,     /*!< FDCAN1 Interrupt line 0                                           */
+  FDCAN1_IT1_IRQn             = 32,     /*!< FDCAN1 Interrupt line 1                                           */
+  FDCAN2_IT0_IRQn             = 33,     /*!< FDCAN2 Interrupt line 0                                           */
+  FDCAN2_IT1_IRQn             = 34      /*!< FDCAN2 Interrupt line 1                                           */
```

Comment:
IRQ to use is 21 and 22.
See Vector table in STM32G0x1 reference manual.

This file comes from https://github.com/STMicroelectronics/STM32CubeG0/blob/master/Drivers/CMSIS/Device/ST/STM32G0xx/Include/stm32g0b1xx.h

Reply:
we do not need this change, do we ?
I'll mark  this as change requested

Reply:
@0xc0170 sorry, I left some pending comments.. 🤦 - please see the above.

Basically, interrupts were defined already, which I missed, but names are different than expected by the can_api and I need some guidance on where these should be unified/mapped. Should we have some extra ifs in can_api, or names should be changed, added or.. don't now what else, but sure there are other options :)

Reply:
Hi
Names in CMSIS/stm32*.h can't be changed,
ST mbed files can_api.c has to be updated to match IRQ name.